client/my-sites/stats/activity-log-banner/upgrade-banner.jsx

```diff
@@ -0,0 +1,62 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { isJetpackSite } from 'state/sites/selectors';
+import Banner from 'components/banner';
+import {
+	PLAN_PERSONAL,
+	PLAN_JETPACK_PERSONAL,
+	FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+	FEATURE_NO_ADS,
+} from 'lib/plans/constants';
+
+class UpgradeBanner extends Component {
+	render() {
+		const { isJetpack, translate } = this.props;
+		return (
+			<div className="activity-log-banner__upgrade">
+				{ isJetpack ? (
+					<Banner
+						callToAction={ translate( 'Get daily backups' ) }
+						dismissPreferenceName="activity-upgrade-banner-jetpack"
+						event="activity_log_upgrade_click_jetpack"
+						feature={ FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY }
+						list={ [
+							translate( 'See all site activity over the past month' ),
+							translate( 'Rewind your site back to any point' ),
+							translate( 'Automatic threat scanning' ),
+						] }
+						plan={ PLAN_JETPACK_PERSONAL }
+						title={ translate( 'Secure your site with daily backups' ) }
+					/>
+				) : (
+					<Banner
+						callToAction={ translate( 'Upgrade' ) }
+						dismissPreferenceName="activity-upgrade-banner-simple"
+						event="activity_log_upgrade_click_wpcom"
+						feature={ FEATURE_NO_ADS }
+						list={ [
+							translate( 'Get a custom domain name' ),
+							translate( 'Remove WordPress.com ads' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Remove WordPress.com Ads' )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activity-log-upgrade-nudges)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).